Resources/Prototypes/Entities/Clothing/Belt/belts.yml

```diff
@@ -608,16 +608,40 @@
         - CartridgeAmmo
 
 - type: entity
-  parent: ClothingBeltSecurity
+  parent: ClothingBeltStorageBase
   id: ClothingBeltSecurityWebbing
-  name: security webbing
+  name: security rig
   description: Unique and versatile chest rig, can hold security gear.
   components:
   - type: Sprite
     sprite: Clothing/Belt/securitywebbing.rsi
   - type: Clothing
     sprite: Clothing/Belt/securitywebbing.rsi
-
+  - type: Storage
+    whitelist:
+      tags:
+        - CigPack
+        - Taser
+        - SecBeltEquip
+        - Radio
+        - Sidearm
+        - MagazinePistol
```

Comment:
make an entity which has this storage component, then make the security rig and security belt inherit from it, this'll fix the bug and reduce copypaste

Reply:
I'll look into it, thanks for the idea